js1/uvod-do-js/cvlekce/email.md

```diff
@@ -1,16 +1,16 @@
 ---
 title: E-mail
 demand: 1
+lead: Vytvořte řetězec obsahující vaši e-mailovou adresu.
+solutionAccess: lock
 ---
 
-Vytvořte řetězec obsahující vaši e-mailovou adresu.
-
-1. Sestavte e-mailovou adresu tak, že sečtete dohromady vaše křestní jméno, znak tečka, vaše příjmení a koncovku `@mujmail.com`.
+Sestavte e-mailovou adresu tak, že sečtete dohromady vaše křestní jméno, znak tečka, vaše příjmení a koncovku `@mujmail.com`.
 
 :::solution
 
 ```js
-'Tom' + '.' + 'Jedno' + '@mujmail.com';
+'Tomáš' + '.' + 'Marný' + '@mujmail.com';
```

Comment:
Myslím, že tam bylo jméno schválně bez háčků a čárek, protože háčky a čárky v mailech nebývají. Takhle je teď, podle mne, méně poznat, že je to e-mailová adresa.

Reply:
Pravda, opravím.